### PR TITLE
Final rewrite (no redux)

### PR DESCRIPTION
**Pull request by khalidwilliams**
@cbdallavalle I took redux out of this final and modified the iterations accordingly. Do you think it's a comparable/appropriate level of difficulty? Last inning we gave students a final without redux, which can be found [here](https://github.com/turingschool-examples/url-shortener-ui) for comparison. 

I'm going to run through the assessment on Monday, and would love to get your initial impressions. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,38 +9,35 @@ Our company is building a prototype ticketing system for a local burrito shop. I
...
-
-Remove Redux from the `App` component and move that functionality to the `Orders` component. Make `App` into a functional component, and bring in the burrito orders directly into the `Orders` component instead.
+Right now the app has a bug somewhere in it. Run the dev server and fix whatever bug is causing the app to crash. 
 
 ### Iteration 2
 
-Add functionality to allow submission of the form when there is at least one ingredient added to the order. If there are no ingredients in the order, you should not be able to submit the order.
-
-Also, right now we're seeing a warning in the console about unique keys... Get rid of that warning.
+Connect the front end to the backend so that when the app loads all existing orders from the server.
```
Comment (cbdallavalle) [inline README.md:22]: so that when the app loads, all existing orders are fetched from the server

Comment (khalidwilliams) [reply README.md]: Fixed!

File: README.md [hunk 2]
```diff
@@ -9,38 +9,35 @@ Our company is building a prototype ticketing system for a local burrito shop. I
...
 
-## Iterations
+## Iterations (new)
 
 ### Iteration 1
 
-Right now the `App` component is connected to the store. While sometimes it can be good to have `App` know about everything, in this case the developer team said that it's not necessary.
-
-Remove Redux from the `App` component and move that functionality to the `Orders` component. Make `App` into a functional component, and bring in the burrito orders directly into the `Orders` component instead.
+Right now the app has a bug somewhere in it. Run the dev server and fix whatever bug is causing the app to crash.
```
Comment (cbdallavalle) [inline README.md:18]: the app has several bugs

**Review by cbdallavalle — APPROVED** [review]
Looks great! I would just add one minor change since we know the app has two bugs in it. I added a comment in Iteration 1. I think it's a GREAT final - nice work!

**Pull request merged**